pyproject.toml

```diff
@@ -540,6 +540,7 @@ markers = [
     "e2e: mark test as an E2E test.",
     "external_sqldialect: mark test as requiring install of an external sql dialect.",
     "filesystem: mark tests using the filesystem as the storage backend.",
+    "mercury: mark tests as using the mercury container locally.",
```

Comment:
Do we need this new test marker?
Can we just run this as part of the `cloud` tests?

Reply:
If we re-use the same `cloud` marker then that makes it trivially for me to update existing cloud tests to also run against this containerized version of mercury (instead of against prod or against a mock) without having to write any new tests.

Reply:
I was worried that we would need to re-create a bunch of saved entities in the existing data context, but it was limited to 2 tests. Not sure why I didn't just try it first! All Cloud tests run against a local version of the API now.